Styles/utilities.css

```diff
@@ -38,6 +38,15 @@
 .flex-spc-arnd {
   justify-content: space-around;
 }
```

Comment:
Give it a meaningful name, this sounds confusing.

Reply:
alright, i'll fix it after completing the current project. thanks for pointing it out